students/eric_rosko/lesson-07/src/add_departments.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+"""
+    Learning persistence with Peewee and sqlite
+    delete the database file to start over
+        (but running this program does not require it)
+
+        populate the DB with data
+"""
+
+from peewee import *
+from v00_personjob_model import Person, Job, Department
+from datetime import datetime
+import logging
+
+def populate_departments():
+    """
+        add job data to database
+    """
+
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
+
+    database = SqliteDatabase('../data/personjob.db')
+
+    logger.info('Working with Department class')
+    logger.info('Creating Department records')
+
+    dept_number = 0
+    dept_name = 1
+    start_date = 2
+    end_date = 3
+
+    departments = [
+        ('A001', 'Mailroom', '2001-09-22', '2003-01-30'),
+        ('B001', 'Sales', '2003-02-01', '2006-10-22')
```

Comment:
I don't believe anything in the assignment mentioned adding start and end date to the db model